### ISSUE TITLE
Improve accessibility of boilerplate

### ISSUE THREAD

**Issue by jasonnrosa** [HubSpot/cms-theme-boilerplate#206]
Improve the overall accessibility of boilerplate/ensure it is accessible (WCAG A-WCAG AA). 

**Current list:**
- [ ] [Make language picker focusable](https://github.com/HubSpot/cms-theme-boilerplate/issues/214)
- [ ] [Add navigation skipper](https://github.com/HubSpot/cms-theme-boilerplate/issues/213)

**Checklist**

- [X] I have read the [CONTRIBUTING](https://github.com/HubSpot/cms-theme-boilerplate/blob/master/CONTRIBUTING.md) document.
- [X] I have checked the [issue tracker](https://github.com/HubSpot/cms-theme-boilerplate/issues) to make sure there isn’t already an open issue.

**Comment by jfudman** [comment]
Not sure where a running list is... but adding a navigation skipper and making the language picker focusable would be a good start. Currently keyboard users cannot focus/change the language picker.

**Comment by jasonnrosa** [comment]
Thanks @jfudman these are great suggestions. I will open up a couple of separate issues for this now.

**Comment by jasonnrosa** [comment]
Going to close in favor of the following issues: 
- https://github.com/HubSpot/cms-theme-boilerplate/issues/226
- https://github.com/HubSpot/cms-theme-boilerplate/issues/225
- https://github.com/HubSpot/cms-theme-boilerplate/issues/224
- https://github.com/HubSpot/cms-theme-boilerplate/issues/214
- https://github.com/HubSpot/cms-theme-boilerplate/issues/213

which are part of a future minor version (progress can be tracked here: https://github.com/HubSpot/cms-theme-boilerplate/projects/5)
